app/src/main/res/layout/recent_patients.xml

```diff
@@ -16,7 +16,7 @@
     style="@style/Clinic.V2.TextAppearance.Subtitle1Center.Grey1"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginBottom="@dimen/spacing_96"
+    android:layout_marginBottom="104dp"
```

Comment:
I die a little inside every time I see a hardcoded value. 😢

Reply:
Ohh but why? This value isn't shared with any screen. What do you suggest?

Reply:
Oh, I just meant in general. :P